doc-content/enterprise-only/getting-started/process-instances-filters-advanced-proc.adoc

```diff
@@ -4,7 +4,7 @@ Use the *Advanced Filters* option to create custom process instance filters. The
 
 .Procedure
 . In {CENTRAL}, go to *Menu* -> *Manage* -> *Process Instances*.
-. On the *Manage Process Instances* page, click *Advanced Filters* icon.
+. On the *Manage Process Instances* page, click *Advanced Filters* icon on the left of the page.
```

Comment:
I would reword:
On the *Manage Process Instances* page, on the left of the page click the  *Advanced Filters* icon .
or
On the *Manage Process Instances* page, on the left of the page click  *Advanced Filters* .
 .

Reply:
Done